src/controllers/genres.js

```diff
@@ -0,0 +1,26 @@
+const {
+    getAllItems,
+    createItem,
+    updateItem,
+    getItemById,
+    deleteItem,
+} = require('./helper-fn-CRUD-operations');
+
+const getAllGenres = (_, res) => getAllItems(res, 'genre');
+
+const createGenre = (req, res) => createItem(res, 'genre', req.body);
+
+const updateGenre = (req, res) =>
```

Comment:
Linter as before. Going to stop writing this now, best bet would be to run it for the whole project